GenderPayGap.WebUI/Views/AddOrganisationSearch/Search.cshtml

```diff
@@ -196,9 +196,9 @@
                             : $"search-result-coho-{organisation.CompanyNumber}";
                         <li id="@(resultId)">
                             <div>
-                                <span class="govuk-!-font-weight-bold">
+                                <h3 class="govuk-heading-s govuk-!-margin-bottom-0">
                                     @(organisation.OrganisationName)
-                                </span>
+                                </h3>
                                 <br/>

```

Comment:
There's now a big gap under each organisation name
If we remove the `<br/>` then it looks good